ui/tests/integration/components/confirm-action-test.js

```diff
@@ -0,0 +1,45 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, click } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import sinon from 'sinon';
+
+module('Integration | Component | confirm-action', function (hooks) {
```

Comment:
Thank you for adding a test!! 😍 🎉

Reply:
Can we add another assertion in either test confirming that the chevron changes directions as expected when **both** `data-test-confirm-button` and `data-test-confirm-cancel-button` are clicked? 

I noticed that on cancel it points down again (which is correct), but on confirm it stayed pointing up. This was in the keymgmt secrets engine if you want to reproduce.
<img width="320" alt="Screen Shot 2022-08-17 at 5 14 52 PM" src="https://user-images.githubusercontent.com/68122737/185265265-466c47ba-0824-4e92-8299-6bd434867e74.png">

Reply:
Good idea. I'll add.